bin/MorphingMSSMvsSM.cpp

```diff
@@ -1344,7 +1348,7 @@ int main(int argc, char **argv) {
     "CMS_htt_boson_res_met_2018",
   };
 
-  for(auto u : met_uncerts) ConvertShapesToLnN (cb.cp().bin_id(mssm_bins), u);
+  for(auto u : met_uncerts) ConvertShapesToLnN (cb.cp().bin_id(mssm_bins).process({"ZTT"}, false), u);
```

Comment:
Why don't we translate these into lnN for ZTT?

Reply:
I just did it this way to be consistent with the embedding which also has a shape uncertainty for the MET related uncertainty. The MC analysis needs more work/checks before we can be satisfied with it so this treatment may get changed eventually.

Reply:
ok, sound good.